website/src/releases/2022-09/readable-page-title.mdx

```diff
@@ -0,0 +1,22 @@
+---
+date: 2022-09-29
+title: Readable page title component
+description: We now have a component that can be used to overwrite the default document's `<title>`, to provide improved support for a Human-readable page title
+type: feature
+topics:
+  - Merchant Center
+  - UI Components
+---
+
+We now have a `Human-readable page title` feature describing as detailed as possible,
+the location or context of the page a user is currently on.
+
+You can now use the [`<ApplicationPageTitle>` component](/api-reference/commercetools-frontend-application-shell#applicationpagetitle) to overwrite the default page title by providing more details about a page by passing an `additionalParts` props,
+an array of strings carrying different descriptions of the features of the page. This provides more readability and context to the page title.
+
+```js
```

Comment:
nit: I wouldn't include the code snippet here since we are already providing links to the the detailed information where readers can have the example code within its context.

Reply:
Thank you for all the feedback.
Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2793/commits/7180e3bc3fd593087432a56d0e48262f57c9fe38